lib/BIGSdb/Application.pm

```diff
@@ -219,6 +219,24 @@ sub _initiate {
 	return;
 }
 
+sub _set_dbconnection_params {
```

Comment:
This needs to be named without the front underscore, i.e. sub set_dbconnection_params, since it is called from outside the module (from Offline::RunJobs), so is not a private method.